plugin/trino-hive/src/test/java/io/trino/plugin/hive/BaseHiveConnectorTest.java

```diff
@@ -597,11 +597,11 @@ public void testInvalidValueForQueryPartitionFilterRequiredSchemas()
     {
         assertQueryFails(
                 "SET SESSION hive.query_partition_filter_required_schemas = ARRAY['tpch', null]",
-                "line 1:1: Invalid null or empty value in query_partition_filter_required_schemas property");
+                "line 1:60: Invalid null or empty value in query_partition_filter_required_schemas property");
```

Comment:
What caused this change?

Reply:
This now points to the position of the offending value, rather than start of the `SET SESSION` (invalid is the value, not the name)

Reply:
And this is due to a node being passed to an exception